kafka-scheduler/src/test/java/com/mesosphere/dcos/kafka/scheduler/RecoverySchedulerTest.java

```diff
@@ -194,23 +227,47 @@ public void testReplaceLastBrokerConstrained() throws Exception {
                 .build();
     }
 
-    private DefaultRecoveryScheduler getTestKafkaRecoveryScheduler() throws Exception {
-        return getTestKafkaRecoveryScheduler(
+    private   List<Protos.TaskStatus> getTestKafkaStatuses(Protos.TaskInfo failedTask,
+                                                                 Collection<Protos.TaskInfo> allTasks) {
+        List<Protos.TaskStatus> statuses = new ArrayList<>();
+        for (Protos.TaskInfo task : allTasks) {
+            if (task.getTaskId().equals(failedTask.getTaskId())) {
+                statuses.add(getDummyBrokerTaskStatusFail(failedTask));
+            } else {
+                statuses.add(getDummyBrokerTaskStatus(task));
+            }
+        }//for
+        return statuses;
+    }
+    private Protos.TaskStatus getDummyBrokerTaskStatus(Protos.TaskInfo taskInfo){
+        return Protos.TaskStatus
+                    .newBuilder().setState(Protos.TaskState.TASK_STAGING).setTaskId(taskInfo.getTaskId()).build();
+    }
+    private Protos.TaskStatus getDummyBrokerTaskStatusFail(Protos.TaskInfo taskInfo){
+         return Protos.TaskStatus.newBuilder()
+                 .setTaskId(taskInfo.getTaskId())
+                 .setState(Protos.TaskState.TASK_FAILED).build();
+    }
+
+    private DefaultPlanScheduler getTestKafkaScheduler(){
+        return new DefaultPlanScheduler(
+                getTestOfferAccepter(),
+                new OfferEvaluator(stateStore),
+                new DefaultTaskKiller(stateStore,failureListener,driver));
+    }
+
+    private DefaultRecoveryPlanManager getTestKafkaRecoveryPlanManager() throws Exception {
+        return getTestKafkaRecoveryPlanManager(
                 new UnconstrainedLaunchConstrainer(),
                 new KafkaFailureMonitor(recoveryConfiguration));
     }
 
-    private DefaultRecoveryScheduler getTestKafkaRecoveryScheduler(
+    private DefaultRecoveryPlanManager getTestKafkaRecoveryPlanManager(
             LaunchConstrainer constrainer,
             FailureMonitor monitor) throws Exception {
-        return new DefaultRecoveryScheduler(
-                stateStore,
-                failureListener,
-                getTestOfferRequirementProvider(),
-                getTestOfferAccepter(),
-                constrainer,
-                monitor,
-                new AtomicReference<>());
+        return new DefaultRecoveryPlanManager(stateStore, getTestOfferRequirementProvider(),
```

Comment:
Move all arguments to their own line for consistency, unless they fit on one line.